std/util/deep_assign.ts

```diff
@@ -24,7 +24,7 @@ export function deepAssign(
       if (typeof target[key] !== `object` || !target[key]) {
         target[key] = {};
       }
-      deepAssign(target[key] as Record<string, unknown>, value);
+      deepAssign(target[key] as Record<string, unknown>, value!);
```

Comment:
`assert(value)` here or `deepAssign` should be accepting `undefined`.

Reply:
So we should be able to remove the not null assert?

Reply:
Yep, my bad :)